### DISCUSSION TITLE
Quick-replace banners

### DISCUSSION THREAD

**Discussion by sph432** [OpenRCT2/OpenRCT2#15773] [Feature requests]
- [x] Reproducible in RCT2 (vanilla)
- [ ] Specific to multiplayer

**Steps to reproduce:**
1. Place a banner on any path.
2. Try to place another in its place.
![20191218_224403](https://user-images.githubusercontent.com/313648/71125846-021ff700-21e8-11ea-9def-7b6cdff2b635.png)

It would be nice to have the existing banner replaced for a reduced price, but keeping its text and no-entry status.

**Comment by ocalhoun6** [comment]
Huh? Why would you want to do this, especially if you want it to remain a no entry banner?

If you want to change the color or the text, why wouldn't you just right-click and edit it?

**Comment by AaronVanGeffen** [comment]
> Huh? Why would you want to do this, especially if you want it to remain a no entry banner?

I take it to mean replacing it with a different banner type — that is, a different piece of scenery. As you probably know, there are many aside from the standard one in the screenshot.
